hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTableResultScanner.java

```diff
@@ -138,7 +149,7 @@ public synchronized Result next() throws IOException {
         return null;
       }
       if (error != null) {
-        FutureUtils.rethrow(error);
+        throw FutureUtils.rethrow(error);
```

Comment:
There's a `throw` inside of the `rethrow` method ; adding one here make this clear and silences a static analysis warning.